src/fragmentarium/ui/info/Details.test.tsx

```diff
@@ -124,6 +124,11 @@ describe('All details', () => {
     expect(screen.getByText(expectedMeasures)).toBeInTheDocument()
   })
 
+  it('Renders editedInOraccProject', () => {
+    const expectedEditedInOraccProject = `${fragment.editedInOraccProject}`
+    expect(screen.getByText(expectedEditedInOraccProject)).toBeInTheDocument()
+  })
```

Comment:
`expect(screen.getByText(fragment.editedInOraccProject)).toBeInTheDocument()` is suffice.

const expectedEditedInOraccProject = `${fragment.editedInOraccProject}` String interpolation `` is not necessary because the editedInOraccProject is already a string. String Interpolation is used if you want to mix it with string like this: `${fragment.editedInOraccProject} hello 123`.

Reply:
Thanks, incorporated